fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/mpjwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2017 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.mpjwtauth;
+
+import org.wildfly.swarm.config.runtime.AttributeDocumentation;
+import org.wildfly.swarm.spi.api.Defaultable;
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static org.wildfly.swarm.spi.api.Defaultable.integer;
+import static org.wildfly.swarm.spi.api.Defaultable.string;
+
+/**
+ * A fraction that adds support for the MicroProfile 1.0 JWT RBAC authentication and authorization spec.
+ */
+@Configurable("swarm.mpjwtauth")
+@DeploymentModule(name = "org.eclipse.microprofile.jwt")
+@DeploymentModule(name = "org.glassfish.javax.json")
+@DeploymentModule(name = "org.jboss.weld.core")
```

Comment:
Why does the deployment need access to Weld core?

Reply:
Should be obsolete. At one point there was use of a core class in the extension, but that was changed.